fake_spectra/absorption.cpp

```diff
@@ -92,12 +92,22 @@ double tophat_kern_frac(double zlow, double zhigh, const double smooth, const do
     return 3./4./M_PI*std::max(0.,zhigh - zlow);
 }
 
+double arepo_kern_frac(double zlow, double zhigh, const double smooth, const double dr2, const double zrange)
```

Comment:
Could you add a comment here like the one about tophat_kern_frac ? Especially it would be nice to explain why you need a factor of unity here rather than 3/4/M_PI.